src/main/webapp/app/exercise/version-history/shared/exercise-version-shared-snapshot-metadata.component.ts

```
@@ -0,0 +1,199 @@
+    private renderMarkdownSections(snapshot: ExerciseSnapshotDTO): void {
+        if (this.plantUmlTimeoutId !== undefined) {
+            clearTimeout(this.plantUmlTimeoutId);
+        }
+        this.injectableCallbacks.length = 0;
+        this.plantUmlWrapper.setExerciseId(snapshot.id);
+
+        this.problemStatement.set(this.renderMarkdown(snapshot.problemStatement));
+        this.gradingInstructions.set(this.renderMarkdown(snapshot.gradingInstructions));
+
+        this.plantUmlTimeoutId = setTimeout(() => {
+            this.injectableCallbacks.forEach((callback) => callback());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)